superset/migrations/versions/fbd55e0f83eb_.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
+
+Revision ID: fbd55e0f83eb
+Revises: ('7467e77870e4', 'de021a1ca60d')
+Create Date: 2018-12-22 17:26:16.113317
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'fbd55e0f83eb'
+down_revision = ('7467e77870e4', 'de021a1ca60d')
```

Comment:
Merge this with the one above?

Reply:
This happens when rebasing multiple times... I'd have to do some manual edits and wait for CI. I'll just run with it as there's not real downside of having an extra empty migration "commit"...